_includes/footer.html

```diff
@@ -108,20 +115,90 @@ <h4 class="text-center"><a href="https://github.com/sugarlabs/www"><b><span clas
     </div>
 </footer>
 
+<!-- Add this CSS to your existing stylesheet or in a style tag -->
+<style>
+@media (max-width: 768px) {
+    .widget {
+        text-align: center;
+        padding: 0 15px;
+    }
+    
+    .leftAlign {
+        padding-left: 0;
+    }
+    
+    .footerHeaderStyle {
+        margin-bottom: 1rem;
+    }
+    
+    .footerListStyle {
+        margin-bottom: 0.5rem;
+    }
+    
+    .grid-two-columns {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 1rem;
+    }
+    
+    #own-ssk {
+        justify-content: center;
+        flex-wrap: wrap;
+        gap: 1rem;
+    }
+    
+    .DonateFooter {
+        margin-top: 1rem;
+    }
+}
+
+@media (max-width: 576px) {
+    .widget {
+        margin-bottom: 2rem;
+    }
+    
+    .ion-icon {
+        font-size: 24px;
+    }
+}
+
+/* Maintain existing styles while adding responsive support */
+.footerListStyle a {
+    color: white;
+    text-decoration: none;
+    transition: color 0.3s ease;
+}
+
+.footerListStyle a:hover {
+    color: #0099FF;
+}
+
+.ion-icon {
+    color: white;
+    margin: 0 8px;
+    transition: color 0.3s ease;
+}
+
+.ion-icon:hover {
+    color: #0099FF;
+}
+</style>
+
 <!-- Javascript files -->
-<script  type="text/javascript" src="{{ site.baseurl }}/js/plugins.js"></script>
-<script  type="text/javascript" src="{{ site.baseurl }}/js/owl.carousel.min.js"></script>
+<script type="text/javascript" src="{{ site.baseurl }}/js/plugins.js"></script>
+<script type="text/javascript" src="{{ site.baseurl }}/js/owl.carousel.min.js"></script>
 <script type="text/javascript">
 if (typeof jQuery.waypoints == 'undefined') {
     document.write('<script src="{{ site.baseurl }}/js/waypoints.js"><\/script>');
 }
 </script>
 
 <!-- Inline Javascript scripts -->
-<script  type="text/javascript">
+<script type="text/javascript">
     $(document).on('click', "a[href='#top'], #scrollToTopBtn", function (e) {
-    $('html, body').animate({ scrollTop: 0 }, 'slow');
-    return false;
+        $('html, body').animate({ scrollTop: 0 }, 'slow');
```

Comment:
While indentation changes are helpful, there are so many in this pull request that it hides the change you are asking for review.  It would be common to use a separate pull request for such changes.